changelog/bugfixes/2022-04-25-fix-dev-container-module-builds.md

```diff
@@ -0,0 +1 @@
+- Added dev-util/pahole to developer container, without it kernel modules built against /usr/src/linux may fail to probe with an 'invalid relocation target' error ([PR#1839](https://github.com/flatcar-linux/coreos-overlay/pull/1839))
```

Comment:
```suggestion
- Added pahole to developer container, without it kernel modules built against /usr/src/linux may fail to probe with an 'invalid relocation target' error ([PR#1839](https://github.com/flatcar-linux/coreos-overlay/pull/1839))
```
I think the dev-util prefix confuses non-Gentoo users (it could be read as an alternative name for pahole)

Reply:
ok, appreciate the feedback, removed the dev-util prefix.